R/util-MakeParamLabels.R

```diff
@@ -0,0 +1,77 @@
+#' Create Labels for Parameters
+#'
+#' @description `r lifecycle::badge("stable")`
+#'
+#'   Convert a vector of parameters to labels in Title Case. `MakeParamLabels`
+#'   adds a `Labels` column to a `data.frame` that has a `Params` column (such
+#'   as `dfGroups`), while `MakeParamLabelsList` returns just the list of named
+#'   parameters.
+#'
+#' @inheritParams shared-params
+#'
+#' @return `dfGroups` with an added `Label` column, or a list of labeled
+#'   parameters.
+#'
+#' @examples
+#' head(gsm::reportingGroups)
+#' MakeParamLabels(head(gsm::reportingGroups))
+#' MakeParamLabels(
+#'   head(gsm::reportingGroups),
+#'   list(ParticipantCount = "Number of Participants")
+#' )
+#' MakeParamLabelsList(head(gsm::reportingGroups$Params))
```

Comment:
tiny typo in this example
```suggestion
#' MakeParamLabelsList(head(gsm::reportingGroups$Param))
```